mission/para_server_init.sqf

```diff
@@ -93,8 +93,8 @@ para_s_bf_respawn_supply_cost = 50;
 
 // Set desired number of simultaneously active zones.
 vn_mf_targetNumberOfActiveZones = 1;
-// Set number of enemies per player.
-para_g_enemiesPerPlayer = 2;
```

Comment:
should be fine. but could cause a noticeable change to gameplay.

Reply:
This is important to look at. Our server cap is 70, the ordinary version is 40 and I dont think I've ever seen more than 80 AI.